### PR TITLE
[Wallet] Redesigned Guides

### PR DESCRIPTION
**Pull request by aaronmgdr**
### Description

Adds Account Key and Celo Gold  Guides

#### Missing
Some Assets are deliberately excluded. 

### Other changes

remove old dollar Education Screen
change the dots used on horizontal Swipers to be dark gray like in mocks. 

### Tested

TBD there is snapshot test 

Screen Shot 2020-06-08 at 2 26 30 PM
Screen Shot 2020-06-08 at 2 26 35 PM
Screen Shot 2020-06-08 at 2 26 40 PM
Screen Shot 2020-06-08 at 2 29 09 PM
Screen Shot 2020-06-08 at 2 29 14 PM
Screen Shot 2020-06-08 at 2 29 18 PM
Screen Shot 2020-06-08 at 2 29 23 PM

### Related issues
#3645 
#3657 
#3884 
#3764 

### Backwards compatibility

nope

### CODE REVIEW

File: packages/mobile/src/account/Education.tsx [hunk 1]
```diff
@@ -1,221 +1,159 @@
...
-export const CTA_CIRCLE_SIZE = 5
+import { TopBarIconButton } from 'src/navigator/TopBarButton.v2'
 
 interface State {
   step: number
 }
 
 interface EducationStep {
-  image: any
+  image: ImageSourcePropType | null
```
Comment (cmcewen) [inline packages/mobile/src/account/Education.tsx:22]: ✨

File: packages/mobile/src/account/GoldEducation.tsx [hunk 2]
```diff
@@ -1,63 +1,83 @@
...
-}
-type Props = DispatchProps
-export class GoldEducation extends React.Component<Props> {
-  static navigationOptions = { header: null }
+export default function GoldEducation() {
+  const { t } = useTranslation(Namespaces.global)
+
+  const dispatch = useDispatch()
+
+  const isCeloEducationComplete = useSelector<RootState, boolean>(
```
Comment (cmcewen) [inline packages/mobile/src/account/GoldEducation.tsx]: there's a typed selector in `src/redux/useSelector`

File: packages/mobile/src/account/GoldEducation.tsx [hunk 3]
```diff
@@ -1,63 +1,83 @@
...
+    (state) => state.goldToken.educationCompleted
+  )
 
-  goToExchange = () => {
-    this.props.setEducationCompleted()
+  const goToExchange = React.useCallback(() => {
```
Comment (cmcewen) [inline packages/mobile/src/account/GoldEducation.tsx]: do we have any conventions around useCallback? i am not sure it's necessary here: https://kentcdodds.com/blog/usememo-and-usecallback

File: packages/mobile/src/navigator/NavigationService.ts [hunk 4]
```diff
@@ -47,6 +47,24 @@ export const replace: SafeNavigate = (...args) => {
     })
 }
 
+// for when a screen should be pushed onto stack even if it already exists in it.
+export const pushToStack: SafeNavigate = (...args) => {
```
Comment (jeanregisser) [inline packages/mobile/src/navigator/NavigationService.ts:51]: 👍

**Review by jeanregisser — APPROVED** [review]
Great work! 👍 🦖 🎉

File: packages/mobile/src/navigator/Navigator.tsx [hunk 5]
```diff
@@ -62,7 +61,7 @@ import {
   nuxNavigationOptions,
   nuxNavigationOptionsNoBackButton,
 } from 'src/navigator/Headers.v2'
-import { navigate, navigateBack } from 'src/navigator/NavigationService'
+import { navigate, navigateBack, navigateToExchangeHome } from 'src/navigator/NavigationService'
```
Comment (tarikbellamine) [inline packages/mobile/src/navigator/Navigator.tsx]:
```suggestion
import {navigateBack, navigateToExchangeHome } from 'src/navigator/NavigationService'
```

**Pull request merged**